images/mediatypes.go

```diff
@@ -57,6 +57,12 @@ const (
 
 	MediaTypeImageLayerEncrypted     = ocispec.MediaTypeImageLayer + "+encrypted"
 	MediaTypeImageLayerGzipEncrypted = ocispec.MediaTypeImageLayerGzip + "+encrypted"
+
+	// OCI media types
+
+	MediaTypeWasmLayerModule    = "application/vnd.w3c.wasm.module.v1+wasm"
+	MediaTypeWasmLayerComponent = "application/vnd.w3c.wasm.component.v1+wasm"
+	MediaTypeWasmLayerConfig    = "application/vnd.wasm.component.config.v1+yaml"
```

Comment:
These are set in stone, I don't know if we would want to add them here or not.

Reply:
conversation happening in https://github.com/containerd/containerd/pull/8699/files#r1233031215 closing this